app/src/main/java/com/simplecity/amp_library/ui/views/multisheet/CustomMultiSheetView.java

```diff
@@ -22,8 +17,8 @@
 
     private static final String TAG = "CustomMultiSheetView";
 
-    @Inject MultiSheetEventRelay multiSheetEventRelay;
-    @Inject MultiSheetSlideEventRelay multiSheetSlideEventRelay;
+    MultiSheetEventRelay multiSheetEventRelay = new MultiSheetEventRelay();
```

Comment:
If we're going to instantiate the `MultiSheetEventRelay` here, we need to make sure we're not instantiating a separate instance elsewhere (which we are doing in `MultiSheedModule`).